simple/src/MediumProblem/SwapArray.java

```diff
@@ -0,0 +1,36 @@
+package MediumProblem;
+
+import java.util.Arrays;
+
+public class SwapArray {
+    int temp;
+    int lastNumber;
+    boolean flag = false;
+    public void endSwap(int arrayOne[]) {
+        // find last value of array
+        lastNumber = arrayOne[arrayOne.length - 1];
+        for (int i = 2; i <= (lastNumber / 2); i++) {
+            if (lastNumber % i == 0) {
+                flag = true;
+                System.out.println("The last Number is :" + lastNumber + "And it is not prime Number");
+                break;
+            }
```

Comment:
Use this block to set the flag only, in not prime case the output will get print twice.